centreon/www/front_src/src/About/translatedLabels.ts

```diff
@@ -0,0 +1,14 @@
+export const labelCentreon = 'Centreon';
+export const labelDevelopedBy = 'Developed by';
+export const labelAnd = 'and';
+export const labelCommunity = 'the community';
+export const labelProjectLeaders = 'Project leaders';
+export const labelDevelopers = 'Developers';
+export const labelContributors = 'Contributors';
+export const labelSecurityAcknowledgement = 'Security acknowledgement';
+export const labelAndManyOthers = 'And many others...';
+export const labelYouCanSeeFullListOnThe = 'You can see the full list on the';

```

Comment:
si le but c'est de concaténer avec "Centreon's GitHub", il faut pas mettre le "the"